### DISCUSSION TITLE
Bootloader not found

### DISCUSSION THREAD

**Discussion by edrocket** [SmartEVSE/SmartEVSE-2#54] [Q&A] [answered]
Hello, I want to load V2.30 into my V2.20 EVSE, downloaded the AN1310 v1.05r. The terminal mode (run app firmware) works, hitting enter shows the EVSE info. But hitting bootloader mode reveals in down left message: Bootloader not found.
Any idea what I should do next?

**Comment by mstegen** [comment]
You can try the following: 
Make sure you set the baudrate to 115200, then press the pause button.
disconnect/power down the SmartEVSE. connect the FTDI cable, and then powerup the SmartEVSE.
now press the bootloader button.

If this does not work, you can send the SmartEVSE back to me, and i'll reprogram it for you.

**Comment by edrocket** [comment]
Thanks for feedback, I'll try. Just read how it should work, so sending a break should cause the micro to reset. Software is not in charging state, I'm sure of that.
I use a PL2303 and I read about issues sending break...

I will figure that out and keep you informed.

**Comment by mstegen** [comment]
Please note that is should be in state A, so not connected to the car. If you want to make sure, just unplug the 12 pin connector.

Not sure if the PL2303 will work. I have never tried it with anything other then a FTDI cable.

**Comment by edrocket** [comment]
Thanks, I have had the controller on my desk, no cables attached :-)
I've found an FTDI cable, will try that at home tonight; I'm sure that will do the trick.

**Comment by edrocket** [comment, marked as answer]
Good news: flashing works. It was the break issue of the PL2303 after all.
Flashed with newly compiled firmware (just, because I can, and to set the master modbus ID to yet another address then 0x01, for my purpose :-))

**Comment by mstegen** [comment]
Good that you got it to work with a FTDI cable!
I'll close this issue now.
